drivers/timer/stm32_lptim_timer.c

```diff
@@ -429,6 +429,35 @@ static int sys_clock_driver_init(void)
 	}
 #endif
 
+#if DT_PROP(DT_DRV_INST(0), st_counter_value)
+	/* Calculate the corresponding prescaler in range [1..128] */
+	uint8_t presci;
+
+	for (presci = 0; presci <= 7 ; presci++) {
+		/* value is in the interval from 0 to 2<<presc */
+		if ((uint32_t)(2 << presci) >=
+			(uint32_t)DT_PROP(DT_DRV_INST(0), st_counter_value)) {
+			break;
+		}
+	}
+
+	/*
+	 * Check if lptim_clock_presc given by DT_PROP(DT_DRV_INST(0), st_prescaler)
+	 * equals the calculated one from the DT_PROP(DT_DRV_INST(0), st_counter_value)
```

Comment:
```
	 * Check if lptim_clock_presc given by DT_INST_PROP(0, st_prescaler)
	 * equals the calculated one from the DT_INST_PROP(0, st_counter_value)
```

Reply:
changed to `DT_INST_PROP(0, st_counter_value)`